### ISSUE TITLE
No keyboard only way to move the find widget

### ISSUE THREAD

**Issue by bpasero** [microsoft/vscode#156251]
Testing #156100

I wonder if I should be able to tab-focus into the grab handle and press arrow keys to be able to move the find widget with keys only?

**Comment by joaomoreno** [comment]
**Verification:**

1. In a tree, press <kbd>Ctrl F</kbd>
2. <kbd>Shift Tab</kbd> to focus the grabber
3. <kbd>LeftArrow</kbd>, <kbd>RightArrow</kbd>, <kbd>Space</kbd> to move the widget around
